spring-integration-jdbc/src/main/java/org/springframework/integration/jdbc/channel/PostgresSubscribableChannel.java

```diff
@@ -75,7 +89,26 @@ public PostgresSubscribableChannel(JdbcChannelMessageStore jdbcChannelMessageSto
 	 */
 	public void setDispatcherExecutor(Executor executor) {
 		Assert.notNull(executor, "An executor must be provided.");
-		this.dispatcher = new UnicastingDispatcher(executor);
+		this.executor = executor;
+	}
+
+	/**
+	 * Sets the transaction manager to use for message processing. Each message will be processed in a
```

Comment:
Imperative language in method Javadocs, please: https://github.com/spring-projects/spring-framework/wiki/Code-Style#javadoc-formatting

Reply:
done